pages/index.tsx

```diff
@@ -117,7 +117,7 @@ export default function Home() {
       <div className="p-8 sm:p-28">
         <div className="font-bold text-lg">Industry</div>
         <div className="max-w-screen-sm pb-16 text-black text-4xl font-bold">
-          <h2>Protecting the data itself to move it from silos towards flexible networks.</h2>
+          <h2>Moving from data silos to flexible networks by securing data constitutionally.</h2>
```

Comment:
not a big fan of constitutionally, doesn't speak to the basic english speaker I am.

Maybe security is not the first argument here, I would rethink this end of this title, we move from silos, to networks where we can share and collaborate on data...